test/org/sagebionetworks/bridge/hibernate/HibernateAccountDaoTest.java

```diff
@@ -723,17 +690,12 @@ public void createAccountSuccess() {
         assertEquals(MOCK_NOW_MILLIS, createdHibernateAccount.getModifiedOn().longValue());
         assertEquals(MOCK_NOW_MILLIS, createdHibernateAccount.getPasswordModifiedOn().longValue());
         assertEquals(AccountStatus.ENABLED, createdHibernateAccount.getStatus());
-
-        // don't call sendEmailVerificationToken
-        verify(mockAccountWorkflowService, never()).sendEmailVerificationToken(any(), any(), any());
     }
 
     @Test
     public void createAccountVerifyEmail() {
-        // Most of this is tested in createAccountSuccess(). Just test email workflow.
-        String accountId = dao.createAccount(STUDY, makeValidGenericAccount(), true);
-        verify(mockAccountWorkflowService).sendEmailVerificationToken(STUDY, accountId, EMAIL);
-
+        dao.createAccount(STUDY, makeValidGenericAccount());
```

Comment:
Now that we don't have verify email in the AccountDao, is this redundant with createAccountSuccess()?

Reply:
Yes it is, removed it.